.erb/configs/webpack.config.renderer.dev.babel.js

```diff
@@ -31,7 +31,7 @@ if (!requiredByDLLConfig && !(fs.existsSync(dllDir) && fs.existsSync(manifest)))
       'The DLL files are missing. Sit back while we build them for you with "yarn build-dll"'
     )
   );
-  execSync('yarn build-dll');
+  execSync('yarn');
```

Comment:
this should be `yarn postinstall`
```suggestion
  execSync('yarn postinstall');
```